Makefile

```diff
@@ -22,3 +22,6 @@ prepare-release: docs build
 synchronize-base-object-schemas:
 	cd weave && make generate_base_object_schemas && \
 	cd ../weave-js && yarn generate-schemas
+
+generate:
```

Comment:
i would call this "generate-bindings" or something that is specific to what you are generating

Reply:
done